tsconfig.json

```diff
@@ -1,15 +1,12 @@
 {
+    "extends": "tsconfigs/browser-module",
     "compilerOptions": {
+        "strictFunctionTypes": false,
+        "strictBindCallApply": false,
+        "noImplicitThis": false,
```

Comment:
If we let tsconfigs' `strict: true` set these implicitly to `true` then there are compilation errors.